src/lib/holocene/select/simple-select.svelte

```diff
@@ -54,7 +54,7 @@
 
 <style lang="postcss">
   select {
-    @apply text-slate-500;
+    @apply border-secondary bg-transparent text-primary;
```

Comment:
Let's make this what we currently have on the `FilterSelect` and that way we can just remove the class on the `FilterSelect` component. 

```suggestion
    @apply border-secondary bg-transparent text-primary outline-none focus-visible:outline focus-visible:outline-blue-700;
```

Reply:
Also, let's make the border `1px` (so replace `border-2` [here](https://github.com/temporalio/ui/blob/main/src/lib/holocene/select/simple-select.svelte#L36) with `border`).